bootstrap/kubeadm/internal/cloudinit/controlplane_join.go

```diff
@@ -54,10 +86,23 @@ func NewJoinControlPlane(input *ControlPlaneJoinInput) ([]byte, error) {
 	// TODO: Consider validating that the correct certificates exist. It is different for external/stacked etcd
 	input.WriteFiles = input.Certificates.AsFiles()
 	input.WriteFiles = append(input.WriteFiles, input.AdditionalFiles...)
+	joinScript, err := generate("JoinControlplaneScript", controlPlaneJoinScript, input)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to generate user data for machine joining control plane")
```

Comment:
"Wrapf" -> "Wrap"

also better to: "JoinControlplaneScript" -> "JoinControlPlaneScript"